src/dd-draggable.ts

```diff
@@ -189,6 +178,11 @@ export class DDDraggable extends DDBaseImplement implements HTMLElementExtendOpt
         this._callDrag(e);
       }
     } else if (Math.abs(e.x - s.x) + Math.abs(e.y - s.y) > 3) {
+      let node = (this.el as GridItemHTMLElement)?.gridstackNode;
+      if (node) {
+        const rect = this.el.getBoundingClientRect();
+        node._originalMousePositionInsideElement = { x: e.clientX - rect.left, y: e.clientY - rect.top };
```

Comment:
techinically this should use s.clientX (starting down event) so you are not 3 pixel + off from where you really clicked. if you move very fast that may be >3 pixels...

Reply:
also I don't like having it above the section comment as if this was special... move it down to where the offset was calculated.